scripts/build/commands/all.js

```diff
@@ -0,0 +1,28 @@
+const path = require(`path`);
+const {
+  webpackBuild,
+  transpile
+} = require(`../../utils/build`);
+const {getAllPackagePaths} = require(`../../utils/package`);
+
+
+module.exports = {
+  command: `all`,
+  desc: `Build all packages`,
+  builder: {},
+  handler: () =>
+    getAllPackagePaths().map((pkg) => {
```

Comment:
AFAI can tell, this actually it doesn't do it in parallel. I think it's because the rest of our build scripts are synchronous. The component packages all get babelified in about 10 sec. The webpack components are the ones that take a really long time.